guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -78,7 +80,7 @@ project.MyClass&nbsp;=&nbsp;function(arg1,&nbsp;arg2)&nbsp;{
 };
 </pre>
 
-<h3 id="frontend-dev-guide-javascript-js-coding-docblock-methodcomments">Method and function comments</h3>
+### Method and function comments
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-methodcomments}` to preserve the custom id.